includes/wikia/upload/WikiaUploadStashFile.class.php

```diff
@@ -39,4 +39,14 @@ public function getThumbUrl( $suffix = false ) {
 		}
 		return $path;
 	}
+
+	/**
+	 * Get an URL to access the original stashed file
+	 *
+	 * @return string
+	 */
+	public function getOriginalFileUrl() {
+		return $this->repo->getZoneUrl( 'temp' ) . '/' . $this->getUrlRel();
```

Comment:
This looks like the original ticket https://wikia-inc.atlassian.net/browse/MWEB-81.